src/ophys_etl/modules/decrosstalk/ophys_plane.py

```diff
@@ -168,6 +179,7 @@ def load_movie_data(self):
         """
         Load the data from self._path; store te data in self._data
         """
+        logger.info(f'loading {self.path}')
```

Comment:
I think this will never speak unless you set the `logLevel` to `INFO` when you create the logger.

Reply:
Defaulting log_level in the schema fixed this problem.

Reply:
I am not sure that this is the same logger as `self.logger` in the `ArgSchemaParser`

Reply:
I can experimentally demonstrate that it does speak now
```
(decrosstalk) [scott.daniel@synapse ~]$ python -m ophys_etl.modules.decrosstalk --input_json decrosstalk_qc_dev/full_input/DECROSSTALK_QUEUE_short.json --output_json short_json.json 
INFO:argschema.argschema_parser:OPHY_ETL_COMMIT_SHA: local build
INFO:argschema.argschema_parser:running on plane: 1071738390
INFO:ophys_etl.modules.decrosstalk.ophys_plane:loading /allen/programs/braintv/production/visualbehavior/prod5/specimen_1046617448/ophys_session_1071648245/ophys_experiment_1071738390/processed/1071738390_suite2p_motion_output.h5
```
though I agree that I am not 100% confident that I know why it speaks

Reply:
ok